app/src/main/java/me/ash/reader/domain/service/GoogleReaderRssService.kt

```diff
@@ -429,7 +429,7 @@ class GoogleReaderRssService @Inject constructor(
                         accountId = accountId,
                         isUnread = unreadIds.contains(articleId),
                         isStarred = starredIds.contains(articleId),
-                        updateAt = it.crawlTimeMsec?.run { Date(this.toLong()) } ?: preDate,
+                        updateAt = it.timestampUsec?.run { Date(this.toLong()/1000) } ?: preDate,
```

Comment:
`timestampUsec` seems better, but can we remove the `/1000` ? We shouldn't need it.